autogpt/autogpt/app/agent_protocol_server.py

```diff
@@ -76,7 +86,7 @@ async def start(self, port: int = 8000, router: APIRouter = base_router):
             "Modified version of The Agent Protocol.",
             version="v0.4",
         )
-
+        create_middleware(app)
```

Comment:
Please just put the middleware thing here, it's quite simple so this will be cleaner:
```suggestion
        @app.middleware("http")
        async def add_custom_header(request: Request, call_next: Callable) -> Response:
            response = await call_next(request)
            response.headers["Content-Type"] = "application/json; charset=utf-8"
            return response
```

Reply:
This should also be added in `forge/agent/agent.py:Agent`, from which `AgentProtocolServer` was derived.

Reply:
it can woks too,may more elegant

Reply:
but , correct it in orge/agent/agent.py:Agent，in func get_agent_app, can it work? maybe you can test again. 
in my case ,it cannot work

Reply:
`forge/agent/agent.py:Agent` is currently broken, don't worry if it doesn't start. @kcze is working on that in #7196. If you can just apply the same fix, that would be great.

Reply:
this is now merged. Resolving